src/v/cluster/members_backend.cc

```diff
@@ -146,6 +146,9 @@ void members_backend::calculate_reallocations(update_meta& meta) {
         // reallocate all partitons for which any of replicas is placed on
```

Comment:
Difference is theres no raft group backing these new types of topics. Maybe something like `raftless_topic` makes more sense?

Reply:
i think your intuition is spot on. the name should be generic enough to handle a large _class_ of types in the future because the last thing we want to do is litter this top level structure with a bunch of type flags.

i'm not going to insist that we change the name, but my opinion is that `non_replicated` is not generic enough.

Reply:
I agree, how does `non_replicatable` sound?